modules/post-installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-configuring-nutanix-failure-domains_{context}"]
+= Adding failure domains to the Infrastructure CRD
+
+You add failure domains to an existing Nutanix cluster by modifying its Infrastructure custom resource definition (CRD) (`infrastructures.config.openshift.io`).
+
```

Comment:
Users also need to modify the ControlPlaneMachineSet resource's spec.template.machines_v1beta1_machine_openshift_io.failureDomains, with a list of the failure domain names the control-plane nodes will spread through.
And if the compute nodes need to spread through the Failure Domains configured in the Infrastructure resource as well, users need to modify or create new MachineSet resources with the properly spread replicas for the Failure Domain consistent providerSpec configuration.

Reply:
Added docs on how to distribute existing control planes and compute machines across failure domains using machine sets. Thank you again @jeana-redhat 

@jeana-redhat will also be updating content on creating compute machine sets and CPMS to cross reference these docs.

Reply:
My PR will not build right until this is merged (the links to Mike's content are not there yet), but it did create a preview for anyone who is interested.
https://github.com/openshift/openshift-docs/pull/70056